.github/workflows/tests.yml

```diff
@@ -82,3 +85,7 @@ jobs:
       - name: Hello World tests
         run: |
           cd hello-world && yarn hardhat compile && yarn run test
+
+      - name: Custom-aa tests
+        run: |
+          cd custom-aa && yarn hardhat compile && yarn run test
```

Comment:
Do we need to run `yarn hardhat compile` if running `yarn test` in turn runs `hardhat test` which performs the compilation anyhow?